interpreter/valid/valid.ml

```diff
@@ -155,8 +155,6 @@ let check_def_type (c : context) (dt : def_type) at =
   | DefFuncT ft -> check_func_type c ft at
   | DefContT ct -> check_cont_type c ct at
 
-
-
 (* Stack typing *)
```

Comment:
```suggestion

(* Stack typing *)
```
Nit: I use the convention to have 2 empty lines before "section header comments" like this.